src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -416,8 +418,7 @@ bool Terminal::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle) noex
         shouldBlink = false;
         break;
     default:
-        finalCursorType = CursorType::Legacy;
-        shouldBlink = false;
+        return true;

```

Comment:
Nit: If we're now treating unrecognised parameters as a no-op in Windows Terminal (which is good), should we not be doing the same thing for conhost in AdaptDispatch?

Reply:
I don’t know how can we debug conhost now. So I intended to keep this. By the way do we want to fix this kind of small bug in conhost, or do we want to keep its behavior. I mean conhost itself is the legacy thing that shouldn’t be changed much for compatibility

Reply:
None of the VT code in conhost is legacy, and this area of the code base is changing all the time. But we can always fix it in a follow-up PR if you don't want to do it now.

Reply:
Thanks. I’m a little uncomfortable that I cannot unittest or manually test conhost. So I don’t feel right about changing that.

I think there’s no need to rush this. And Dustin is not available the next several days. I’m OK waiting for more voices and comments

Reply:
If you look in the _Conhost_ folder of the of the OpenConsole solution, there's a _Host.EXE_ project that you can run to debug conhost. In the _Debugging_ section of the properties, the _Command Arguments_ specify what shell to use - cmd.exe, bash.exe, etc.

As for the conhost unit testing, it'll depend on what exactly you're doing, but often that will be handled in [ScreenBufferTests.cpp](/microsoft/terminal/blob/master/src/host/ut_host/ScreenBufferTests.cpp) and/or [adapterTest.cpp](/microsoft/terminal/blob/master/src/terminal/adapter/ut_adapter/adapterTest.cpp).